template_scripts/04_install_qubes.sh

```diff
@@ -63,6 +63,16 @@ if [ "0$TEMPLATE_ROOT_WITH_PARTITIONS" -eq 1 ]; then
     chroot_cmd umount /sys /dev
 fi
 
+# xinitrc script
+if [ "${DIST/fc/}" -ge 25 ]; then
+    case "$TEMPLATE_FLAVOR" in
+      "xfce")
+          echo "--> Setting up XFCE xinitrc script..."
+          cp ${SCRIPTSDIR}/extras/50-xfce-desktop.sh ${INSTALLDIR}/etc/X11/xinit/xinitrc.d
```

Comment:
Better package such stuff somewhere, instead of copying during template build. Otherwise there is no way to update such file in a sane way. I think this belongs to gui-agent-linux/appvm-scripts/etc/X11/xinit/xinitrc.d. Can be either packaged as a separate one-file sub-package, or enabled with some configuration.

Reply:
Thank you for your feedback. This is what I thought about but before modifying something in other packages I wanted some feedback from someone else. I will prepare it.